front/pages/w/[wId]/builder/data-sources/managed.tsx

```diff
@@ -158,6 +211,11 @@ export const getServerSideProps = withDefaultUserAuthRequirements<{
             connector: null,
             fetchConnectorError: true,
             fetchConnectorErrorMessage: statusRes.error.message,
+            editedByUser: getUserImageUrl(mds),
+            usage: await getDataSourceUsage({
```

Comment:
Is it okay to compute usage for all data sources like folders and websites? Isn't it too slow?

Reply:
Will definitely test it on front edge and create a filter if it is 👍🏼

Reply:
Looks alright, will add a ticket anyways 👍🏼